include/faunus/space.h

```diff
@@ -316,6 +368,7 @@ namespace Faunus
           for ( auto mol : molecule )
               while ( mol.Ninit-- > 0 )
                   insert(mol.id, mol.getRandomConformation(geo, p));
+          molTrack.fillIndex();
```

Comment:
Could you not just use the original tracker and here fill it without calling a member function?